ui/src/components/Nav/Desktop/index.tsx

```diff
@@ -39,27 +39,38 @@ const Desktop: React.FC<Props> = (props): React.ReactElement => (
-                                            <HeadlessUI.Menu.Item>
-                                                <div>
-                                                    {subItem?.label && subItem.value ? (
-                                                        <Components.Link
-                                                            href={subItem.value}
-                                                            className="block w-full rounded border-transparent outline-0 focus:ring-2 focus:ring-yellow-400"
+                                    {item.subItems.map((subItem, index) => {
+                                        return subItem?.label && subItem.value ? (
+                                            <li className="py-2 px-3 text-teal-600 transition-colors duration-500 dark:text-white-50">
+                                                <HeadlessUI.Menu.Item as="a" key={index} href={subItem.value}>
+                                                    {({ active }) => (
+                                                        <span
+                                                            className={`${
+                                                                active ? 'border-yellow-400' : 'border-transparent'
+                                                            } text-white m-0 block w-full rounded-md border-2 p-0.5`}
                                                         >
-                                                            <span className="">{subItem.label}</span>
-                                                        </Components.Link>
-                                                    ) : (
-                                                        subItem
+                                                            {subItem.label}
+                                                        </span>
                                                     )}
-                                                </div>
-                                            </HeadlessUI.Menu.Item>
-                                        </li>
-                                    ))}
+                                                </HeadlessUI.Menu.Item>
+                                            </li>
+                                        ) : (
+                                            <li className="py-2 px-3 text-teal-600 transition-colors duration-500 dark:text-white-50">
+                                                <HeadlessUI.Menu.Item key={index}>
+                                                    {({ active }) => (
+                                                        <button
+                                                            onClick={props.handleLogout}
+                                                            className={`${
+                                                                active ? 'border-yellow-400' : 'border-transparent'
```

Comment:
The only downside with this is if the user has their mouse pointer over where the dropdown becomes present, the border will become transparent due to the hover effect.

[![Image from Gyazo](https://i.gyazo.com/50c0af1e1acbb74ddf35b35f11d319b1.gif)](https://gyazo.com/50c0af1e1acbb74ddf35b35f11d319b1)

Reply:
Sorry scratch my previous comment (now deleted) - my bad,  good spot. I'll play about with the css and see if i can get the border to show

Reply:
Unfortunately I think headlessUI ties the 'active' state to components that the user hovers over so putting an important option on the active yellow border utility doesn't work and I'm a bit stumped. I can't think of any other way to suppress the yellow border without the above drawback occurring. 

It might be worth having a convo with UX/ product to see if they'd prefer the consistency of having no border on hover in the dropdown menu with the drawback you mentioned or just going with the border on hover? Personally, I'm not sure 🤔